[MTM-52611] update alarms announcement (#1997)
SoftwareAG/c8y-docs: content/change-logs/application-enablement/ui-c8y-1020-0-0-alarms-redesign-and-improvements.md

---
In the upcoming version, a redesign of the alarms feature will be introduced with both visual and functional enhancements.

**Breaking change**

> * The AngularJS widget implementation is no longer available, to use the alarms widgets in a custom application, users will need to import the `AlarmsWidgetModule` from `@c8y/ngx-components/widgets/implementations/alarms`. To use the new alarms view only, users will need to import the `AlarmsModule` from `@c8y/ngx-components/alarms`.

Reviewer: ```suggestion
* The AngularJS widget implementation is no longer available. To use the "Alarms" widgets in a custom application, users must import the `AlarmsWidgetModule` from `@c8y/ngx-components/widgets/implementations/alarms`. To use the new alarms view only, users must import the `AlarmsModule` from `@c8y/ngx-components/alarms`.
```
